include/cvc5/cvc5_proof_rule.h

```diff
@@ -1351,6 +1351,55 @@ enum ENUM(ProofRule)
    * \endverbatim
    */
   EVALUE(QUANT_VAR_REORDERING),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Quantifiers -- Exists string length**
+   *
+   * .. math::
+   *   \inferrule{-\mid T n i} {\mathit{len}(k) = n}
+   *
+   * where :math:`k` is a skolem of string or sequence type :math:`T` and
+   * :math:`n` is a non-negative integer. The argument :math:`i` is an
+   * identifier for :math:`k`. These three arguments are the indices of
+   * :math:`k`, whose skolem identifier is
+   * :cpp:enumerator:`WITNESS_STRING_LENGTH <cvc5::SkolemId::WITNESS_STRING_LENGTH>`.
+   *
+   * \endverbatim
+   */
+  EVALUE(EXISTS_STRING_LENGTH),
```

Comment:
Isn't it better to add these rules afterwards, together with the infrastructure to how they will be used?